docs/proposals/push-metrics.md

```diff
@@ -17,7 +17,7 @@ provide metrics through Heapster.
 ### Target Audience and Metrics
 
 Like the existing custom metrics pull mechanism, the metrics pushed through the
-push mechanism are intended to be those metrics that are useful for consumption
+push mechanism are intended to be useful for consumption
```

Comment:
you've changed the meaning of the sentence.  Whereas earlier the sentence implied that we intended that you only expose useful metrics for collection, the new sentence says that we intend all metrics exposed to be useful.  In the earlier sentence, the intention is that metrics have usefulness coming into Heapster, but with your change the metrics are granted inherent usefulness coming out of Heapster, having none to begin with.